mix.exs

```diff
@@ -4,8 +4,8 @@ defmodule AwsSsmProvider.MixProject do
   def project do
     [
       app: :aws_ssm_provider,
-      version: "0.2.1",
-      elixir: "~> 1.7",
+      version: "0.3.0",
+      elixir: "~> 1.8",
```

Comment:
@erikosmond Elixir has moved on to 1.9. I suggest we bump this to 1.8 here so we don't feel the need to keep testing against old versions of Elixir, but I'll revert if you don't agree.